src/generic/listctrl.cpp

```diff
@@ -1932,6 +1935,9 @@ void wxListMainWindow::RefreshLine( size_t line )
             return;
     }
 
+    if ( line == (size_t)-1 )
+        return;
+
```

Comment:
Same question as above: how do we end up refreshing an invalid line?

Reply:
Sorry, this is also due to my changes, fixed now.
[Here](https://github.com/wxWidgets/wxWidgets/blob/master/src/generic/listctrl.cpp#L2584#L2638), `oldCurrent` could be invalid because `m_current` is only initialised when an item is selected or the first time the list control gain focus.